dartsim/src/SDFFeatures.cc

```diff
@@ -608,6 +608,31 @@ Identity SDFFeatures::ConstructSdfLink(
 
   bodyProperties.mInertia.setLocalCOM(localCom);
 
+  // If added mass is provided, add that to DART's computed spatial tensor
+  // TODO(chapulina) Put in another feature
+  if (sdfInertia.FluidAddedMass().has_value())
+  {
+    // TODO check off-diagonal blocks
```

Comment:
I would comment here about how the order of variables differs between SDFormat / Gazebo and Dartsim / Featherstone, which is why the submatrices need to be shuffled.